data/json/vehicleparts/combustion.json

```diff
@@ -36,6 +36,15 @@
     "//": "30% energy efficiency",
     "description": "A combustion engine.  Burns gasoline fuel from a tank in the vehicle."
   },
+  {
+    "abstract": "gas_turbine_engine",
+    "copy-from": "engine_combustion",
+    "type": "vehicle_part",
+    "fuel_type": "gasoline",
```

Comment:
jp8 (the default fuel for these things) is closer to diesel than gasoline, but has an energy density closer to gasoline.  I could go either way on the default fuel.